src/Compilers/CSharp/Test/Semantic/Semantics/NameOfTests.cs

```diff
@@ -1343,5 +1343,51 @@ private static void M(in string value)
     }
 }", expectedOutput: "Main");
         }
+
+        [Fact, WorkItem(20600, "https://github.com/dotnet/roslyn/issues/20600")]
+        public void PermitInstanceQualifiedFromType()
```

Comment:
>PermitInstanceQualifiedFromType [](start = 20, length = 31)

Please add similar test for VB.

Reply:
Sure, but unless VB's spec has changed since the LDM meetings (like C#'s spec has changed), these cases will result in an error at compile-time.